stock_storage_type_putaway_abc/models/product_template.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+from .stock_location import ABC_SELECTION
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = "product.template"
+
+    abc_storage = fields.Selection(ABC_SELECTION, default="b")
```

Comment:
This should be put in a new module. Too much dependencies pulled if we want just that one.